docs/v0.2.0-beta.7/quickstart.md

```diff
@@ -253,16 +253,16 @@ Shorthands make writing your server definition concise, so you should use them w
 
 ## Passthrough
 
-Mirage is a great tool to use even if you're working on an existing app that doesn't have any mocks. By default, Mirage throws an error if your Ember app makes a request that doesn't have a corresponding mock defined. To avoid this, tell Mirage to let unhandled requests pass through:
+Mirage is a great tool to use even if you don't want to fake your entire API. By default, Mirage throws an error if your Ember app makes a request that doesn't have a corresponding route handler defined. To avoid this, tell Mirage to let unhandled requests pass through:
```

Comment:
how about 

"... even if you're working on an existing app, or if you don't want to fake your entire API."

I've heard folks say before that they can't use Mirage since they're not starting a new app, so I just want to be explicit here